avocado/core/main.py

```diff
@@ -51,13 +52,18 @@ def handle_exception(*exc_info):
     tmp, name = tempfile.mkstemp(".log", prefix, get_crash_dir())
     os.write(tmp, msg.encode('utf-8'))
     os.close(tmp)
-    # Print friendly message in console-like output
-    msg = ("Avocado crashed unexpectedly: %s\nYou can find details in %s\n"
-           % (exc_info[1], name))
+    if exc_info[0] == KeyboardInterrupt:
```

Comment:
Nitpick: identity comparisons in this case are preferred to equality.

Reply:
@clebergnu, Fixed with a force-push.